.github/workflows/formatting.yaml

```diff
@@ -0,0 +1,27 @@
+name: check Python formatting
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
You want to be using the standard DM formatting workflow.

https://github.com/lsst/daf_butler/blob/main/.github/workflows/formatting.yaml